### DISCUSSION TITLE
Applications that cannot switch focus in full screen mode

### DISCUSSION THREAD

**Discussion by lorre0322** [siduck/chadwm#184] [Q&A]
How to switch applications in full screen mode, so that the full screen applications also change instead of staying still, which is meaningless.

Whatever,it's a nice dwm！wish you have a nice day ！😘

**Comment by siduck** [comment]
did u try super + f?

**Reply by lorre0322** [reply]
yep！

**Reply by siduck** [reply]
@lorre0322 super f should work! or try super m
